src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -236,8 +236,10 @@ class LRCallbackBase(PhaseCallback):
 
     def __init__(self, phase, initial_lr, update_param_groups, train_loader_len, net, training_params, **kwargs):
         super(LRCallbackBase, self).__init__(phase)
+        if not isinstance(initial_lr, dict):
+            initial_lr = {"default": float(initial_lr)}
```

Comment:
I think it make sense to introduce `DEFAULT_LEARNING_RATE_KEY` variable with value of `default` (Or `_default_` see https://github.com/Deci-AI/super-gradients/pull/1612/files#r1386196047)
The goal is to minimize usage of hard-coded string constants

Reply:
I kind of feel this is a bit of an overkill to have an environment variable to for "default" string...
I will add it if you  insist on it.

Reply:
this is not meant to be environment variable , rather a plan variable: `DEFAULT_LEARNING_RATE_KEY = "default"` used solely for purpose of not placing "default" string constant in many places and use `DEFAULT_LEARNING_RATE_KEY` instead in the code (In YAML files is still would be "default" anyway)
No hard feelings in having it thought